CI.md

```diff
@@ -125,8 +125,26 @@ Service Account shall have following permissions ([IAM roles](https://cloud.goog
 | Java Wordcount Direct Runner | Runs Java WordCount example with Direct Runner.                                               | Yes              | Yes                   | Yes           | -                        |
 | Java Wordcount Dataflow      | Runs Java WordCount example with DataFlow Runner.                                             | -                | Yes                   | Yes           | Yes                      |
 
+### All migrated workflows run based on the following triggers
+
+| Description | Pull Request Run | Direct Push/Merge Run | Scheduled Run | Workflow Dispatch |
+|-------------|------------------|-----------------------|---------------|-------------------|
+| PostCommit  | No               | Yes                   | Yes           | Yes               |
+| PreCommit   | Yes              | Yes                   | Yes           | No                |              
```

Comment:
Is there a reason not to include workflow dispatch on the precommits?

Reply:
Oh, at the moment there is no way to implement trigger phrases on postcommits, we decided to emulate the behavior using workflow_dispatch; and, since precommits get executed on commits we didn't deem it necessary to include workflow_dispatch on them.
Should we include them also on the precommits?

Reply:
Yeah, I'd say lets include it - its nice to be able to trigger jobs like that arbitrarily on master or other branches if you're tracking down a flake

Reply:
Sounds like a plan!
We'll include workflow_dispatch for precommits also.
Thanks for the feedback Danny.